### DISCUSSION TITLE
Polish special characters on Godot Engine.

### DISCUSSION THREAD

**Discussion by Piotrek94K** [godotengine/godot-proposals#5814] [Engine Core]
Hi. Today, while creating my game's menu, I noticed that Godot doesn't support Polish characters like 'ż' 'ć' etc. Is it possible to add their support in one of the updates? Thank you very much for your time.

**Comment by Calinou** [comment]
You need to use a custom font. The default project font only features basic Latin-1 support to keep binary size low. The font is embedded into all exported projects, so it needs to be kept as small as possible.

See also https://github.com/godotengine/godot-proposals/issues/5779.

**Reply by Piotrek94K** [reply]
I understand, thank you very much.
